Make failed: PVR_CHANNEL has no member named 'strStreamURL'

Log:

```
.../pvr.iptvsimple/src/PVRIptvData.cpp: In member function 'virtual PVR_ERROR PVRIptvData::GetChannels(ADDON_HANDLE, bool)':
.../pvr.iptvsimple/src/PVRIptvData.cpp:583:27: error: 'PVR_CHANNEL {aka struct PVR_CHANNEL}' has no member named 'strStreamURL'
       strncpy(xbmcChannel.strStreamURL, channel.strStreamURL.c_str(), sizeof(xbmcChannel.strStreamURL) - 1);
                           ^
.../pvr.iptvsimple/src/PVRIptvData.cpp:583:90: error: 'PVR_CHANNEL {aka struct PVR_CHANNEL}' has no member named 'strStreamURL'
       strncpy(xbmcChannel.strStreamURL, channel.strStreamURL.c_str(), sizeof(xbmcChannel.strStreamURL) - 1);
                                                                                          ^
CMakeFiles/pvr.iptvsimple.dir/build.make:86: recipe for target 'CMakeFiles/pvr.iptvsimple.dir/src/PVRIptvData.cpp.o' failed
make[5]: *** [CMakeFiles/pvr.iptvsimple.dir/src/PVRIptvData.cpp.o] Error 1
CMakeFiles/Makefile2:99: recipe for target 'CMakeFiles/pvr.iptvsimple.dir/all' failed
make[4]: *** [CMakeFiles/pvr.iptvsimple.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make[3]: *** [all] Error 2
CMakeFiles/pvr.iptvsimple.dir/build.make:112: recipe for target 'pvr.iptvsimple-prefix/src/pvr.iptvsimple-stamp/pvr.iptvsimple-build' failed
make[2]: *** [pvr.iptvsimple-prefix/src/pvr.iptvsimple-stamp/pvr.iptvsimple-build] Error 2
CMakeFiles/Makefile2:164: recipe for target 'CMakeFiles/pvr.iptvsimple.dir/all' failed
make[1]: *** [CMakeFiles/pvr.iptvsimple.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/pvr.iptvsimple/addon.xml.in
+++ b/pvr.iptvsimple/addon.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.iptvsimple"
-  version="3.1.1"
+  version="3.2.0"
   name="PVR IPTV Simple Client"
   provider-name="nightik">
   <requires>@ADDON_DEPENDS@</requires>

--- a/pvr.iptvsimple/changelog.txt
+++ b/pvr.iptvsimple/changelog.txt
@@ -1,3 +1,6 @@
+v3.2.0
+- Updated to PVR addon API v5.5.0
+
 v3.1.1
 - removed line length limitation when parsing M3U files
 

--- a/src/PVRIptvData.cpp
+++ b/src/PVRIptvData.cpp
@@ -579,7 +579,6 @@ PVR_ERROR PVRIptvData::GetChannels(ADDON_HANDLE handle, bool bRadio)
       xbmcChannel.bIsRadio          = channel.bRadio;
       xbmcChannel.iChannelNumber    = channel.iChannelNumber;
       strncpy(xbmcChannel.strChannelName, channel.strChannelName.c_str(), sizeof(xbmcChannel.strChannelName) - 1);
-      strncpy(xbmcChannel.strStreamURL, channel.strStreamURL.c_str(), sizeof(xbmcChannel.strStreamURL) - 1);
       xbmcChannel.iEncryptionSystem = channel.iEncryptionSystem;
       strncpy(xbmcChannel.strIconPath, channel.strLogoPath.c_str(), sizeof(xbmcChannel.strIconPath) - 1);
       xbmcChannel.bIsHidden         = false;

--- a/src/client.cpp
+++ b/src/client.cpp
@@ -457,4 +457,5 @@ PVR_ERROR DeleteAllRecordingsFromTrash() { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR SetEPGTimeFrame(int) { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR GetDescrambleInfo(PVR_DESCRAMBLE_INFO*) { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR SetRecordingLifetime(const PVR_RECORDING*) { return PVR_ERROR_NOT_IMPLEMENTED; }
+PVR_ERROR GetStreamTimes(PVR_STREAM_TIMES *times) { return PVR_ERROR_NOT_IMPLEMENTED; }
 }

--- a/src/client.cpp
+++ b/src/client.cpp
@@ -443,7 +443,6 @@ PVR_ERROR DeleteTimer(const PVR_TIMER &timer, bool bForceDelete) { return PVR_ER
 PVR_ERROR UpdateTimer(const PVR_TIMER &timer) { return PVR_ERROR_NOT_IMPLEMENTED; }
 void DemuxAbort(void) {}
 DemuxPacket* DemuxRead(void) { return NULL; }
-unsigned int GetChannelSwitchDelay(void) { return 0; }
 bool IsTimeshifting(void) { return false; }
 bool IsRealTimeStream(void) { return true; }
 void PauseStream(bool bPaused) {}

--- a/src/client.cpp
+++ b/src/client.cpp
@@ -36,7 +36,6 @@ using namespace ADDON;
 bool           m_bCreated       = false;
 ADDON_STATUS   m_CurStatus      = ADDON_STATUS_UNKNOWN;
 PVRIptvData   *m_data           = NULL;
-bool           m_bIsPlaying     = false;
 PVRIptvChannel m_currentChannel;
 
 /* User adjustable settings are saved here.
@@ -340,32 +339,12 @@ PVR_ERROR GetChannels(ADDON_HANDLE handle, bool bRadio)
   return PVR_ERROR_SERVER_ERROR;
 }
 
-bool OpenLiveStream(const PVR_CHANNEL &channel)
+const char *GetLiveStreamURL(const PVR_CHANNEL &channel)
 {
-  if (m_data)
-  {
-    CloseLiveStream();
+  if (m_data && m_data->GetChannel(channel, m_currentChannel))
+    return m_currentChannel.strStreamURL.c_str();
 
-    if (m_data->GetChannel(channel, m_currentChannel))
-    {
-      m_bIsPlaying = true;
-      return true;
-    }
-  }
-
-  return false;
-}
-
-void CloseLiveStream(void)
-{
-  m_bIsPlaying = false;
-}
-
-bool SwitchChannel(const PVR_CHANNEL &channel)
-{
-  CloseLiveStream();
-
-  return OpenLiveStream(channel);
+  return "";
 }
 
 PVR_ERROR GetStreamProperties(PVR_STREAM_PROPERTIES* pProperties)
@@ -406,7 +385,6 @@ PVR_ERROR SignalStatus(PVR_SIGNAL_STATUS &signalStatus)
 }
 
 /** UNUSED API FUNCTIONS */
-const char * GetLiveStreamURL(const PVR_CHANNEL &channel)  { return ""; }
 bool CanPauseStream(void) { return false; }
 int GetRecordingsAmount(bool deleted) { return -1; }
 PVR_ERROR GetRecordings(ADDON_HANDLE handle, bool deleted) { return PVR_ERROR_NOT_IMPLEMENTED; }
@@ -417,7 +395,9 @@ PVR_ERROR RenameChannel(const PVR_CHANNEL &channel) { return PVR_ERROR_NOT_IMPLE
 PVR_ERROR MoveChannel(const PVR_CHANNEL &channel) { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR OpenDialogChannelSettings(const PVR_CHANNEL &channel) { return PVR_ERROR_NOT_IMPLEMENTED; }
 PVR_ERROR OpenDialogChannelAdd(const PVR_CHANNEL &channel) { return PVR_ERROR_NOT_IMPLEMENTED; }
+void CloseLiveStream(void) { }
 bool OpenRecordedStream(const PVR_RECORDING &recording) { return false; }
+bool OpenLiveStream(const PVR_CHANNEL &channel) { return false; }
 void CloseRecordedStream(void) {}
 int ReadRecordedStream(unsigned char *pBuffer, unsigned int iBufferSize) { return 0; }
 long long SeekRecordedStream(long long iPosition, int iWhence /* = SEEK_SET */) { return 0; }

